src/Standard.Licensing/LicenseAttributes.cs

```diff
@@ -34,16 +34,16 @@ namespace Standard.Licensing
     /// </summary>
     public class LicenseAttributes
     {
-        protected readonly XElement xmlData;
-        protected readonly XName childName;
+        protected readonly XElement XmlData;
+        protected readonly XName ChildName;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="LicenseAttributes"/> class.
         /// </summary>
         internal LicenseAttributes(XElement xmlData, XName childName)
         {
-            this.xmlData = xmlData ?? new XElement("null");
-            this.childName = childName;
+            XmlData = xmlData ?? new XElement("null");
+            ChildName = childName;
```

Comment:
This change moves away from the coding style in the project, but with that being said, I believe most coding standards would say that rather than being PascalCase, private and protected members should be with a camelCase with a leading _